lib/vanagon/platform.rb

```diff
@@ -221,11 +221,22 @@ def is_cisco_wrlinux?
       return !!@name.match(/^cisco-wrlinux-.*$/)
     end
 
-    # Utility matcher to determine is the platform is an osx variety
+    # Utility matcher to determine if the platform is an osx variety
     #
+    # @deprecated Please use is_macos? instead
     # @return [true, false] true if it is an osx variety, false otherwise
     def is_osx?
-      return !!@name.match(/^osx-.*$/)
+      warn "is_osx? is a deprecated method, please use is_macos? instead."
+      return true if is_macos?
+      return true if @name =~ /^osx-.*$/
```

Comment:
Are people going to see the deprecation warning and try to use `is_macos` when the platform is still osx something something? That sounds like something I would do, and this would break that.